twitter/View Controllers/TimelineViewController.m

```diff
@@ -85,12 +90,9 @@ - (nonnull UITableViewCell *)tableView:(nonnull UITableView *)tableView
     cell.retweetsCountLabel.text = [NSString stringWithFormat:@"%d", cell.tweet.retweetCount];
     cell.favoritesCountLabel.text = [NSString stringWithFormat:@"%d", cell.tweet.favoriteCount];
     cell.createdAtLabel.text = cell.tweet.createdAtString;
-    
-    NSURL *profileImageURL = [NSURL URLWithString:cell.tweet.user.profileImageURLString];
-    [cell.profileImageView setImageWithURL:profileImageURL];
+    [cell.profileImageView setImageWithURL:cell.tweet.user.profileImageURL];
```

Comment:
make tweet a local variable so you don't have to reach back into the place you just set it. Not a big deal but it's harder to think about this way.